### PR TITLE
provide a scenario tester to run through actions on a host in sequenc…

### PR DESCRIPTION
**Pull request by shibbas**
…e and parallelly. Enables creating simulated test loads.

### CODE REVIEW

File: Tests/ScenarioTester/ScenarioTester.nuspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <iconUrl>https://raw.githubusercontent.com/wiki/Microsoft/data-accelerator/tutorials/images/roundwhite6464.PNG</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Enables running scenarios against a server or application.</description>
+    <copyright>� Microsoft Corporation. All rights reserved.</copyright>
+    <tags>ScenarioTester</tags>
+    <dependencies>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="..\applicationpackage\*.sfpkg" target="lib" />
```
Comment (carlbrochu) [inline Tests/ScenarioTester/ScenarioTester.nuspec]: do you have any service fabric services in scenariotester?  if not you do not need this.

File: Tests/ScenarioTester/ScenarioTester/Request.cs [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+// *********************************************************************
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+using static System.FormattableString;
+
+namespace Microsoft.TestCommon.Utilities.ScenarioTester
```
Comment (carlbrochu) [inline Tests/ScenarioTester/ScenarioTester/Request.cs]: Can we avoid putting the Microsoft name in the namespace?

File: Tests/ScenarioTester/ScenarioTester.nuspec (hunk 1 above)
Comment (shibbas) [reply Tests/ScenarioTester/ScenarioTester.nuspec]: Removed

File: Tests/ScenarioTester/ScenarioTester/Request.cs (hunk 2 above)
Comment (shibbas) [reply Tests/ScenarioTester/ScenarioTester/Request.cs]: Fixed.

File: Tests/ScenarioTester/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,203 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+############################### 
+# Core EditorConfig Options   #
```
Comment (s-tuli) [inline Tests/ScenarioTester/.editorconfig]: Great that you added .editorconfig. Is this editorconfig the same as the version under: https://github.com/microsoft/data-accelerator/blob/master/Services/.editorconfig?

Comment (shibbas) [reply Tests/ScenarioTester/.editorconfig]: yes, it is the same.

**Comment by shibbas** [comment]
Thanks for the review!

**Pull request merged**